mods/persistence/controllers/subsystems/initialization/chargen.dm

```diff
@@ -28,16 +27,99 @@ SUBSYSTEM_DEF(chargen)
 				CHECK_TICK
 
 /datum/controller/subsystem/chargen/proc/get_spawn_turf()
-	return pick(chargen_spawns)
+	//Grab first free pod
+	for(var/area/chargen/A in chargen_areas)
+		if(chargen_areas[A] > 0)
+			continue
+		var/obj/abstract/landmark/chargen_spawn/L = locate(/obj/abstract/landmark/chargen_spawn) in A
```

Comment:
I think this proc is extraneous anyway, but you should be able to get the landmark from A.chargen_landmark

Reply:
Yeah, removed the proc now, so its fine.